lib/facter/fpm_pools.rb

```diff
@@ -0,0 +1,9 @@
+Facter.add(:fpm_pools) do
+  confine :osfamily => 'Debian'
+
+  setcode do
+    Dir.glob('/etc/php5/fpm/pool.d/*.conf').each do |pool_config|
```

Comment:
it will not work on RetHat distros as set on params.pp, the fpm configuration leaves at:
/etc/php-fpm.d/